src/components/BookItem.js

```diff
@@ -11,9 +11,9 @@ const BookItem = ({
       <h1>{title}</h1>
       <p>{author}</p>
       <div className="meta">
-        <button type="button">Comments</button>
+        <button className="button-unique" type="button">Comments</button>
         <Button text="Remove" purpose={removeBook} id={itemId} />
-        <button type="button">Edit</button>
+        <button className="button-unique" type="button">Edit</button>
       </div>
 
```

Comment:
- [ ] Adjust the fonts of the number percentage and look on design it should be on the middle of the circle

Yours:
![image](https://github.com/0sugo/bookstore/assets/62705947/94adabaf-0e5e-4f08-9f99-3c5512530b4f)
Design:
![image](https://github.com/0sugo/bookstore/assets/62705947/951a8e3e-3d21-49ce-8bfa-852b7f3a5eb2)